cypress/integration/common/index.js

```diff
@@ -170,6 +170,18 @@ And(
 		cy.get(`a[href="${linkHref}"]`).should('have.text', linkText);
 	}
 );
+
+/*
+    -----------------------
+        Manual Page results
+    -----------------------
+*/
+And('the page goes into mobile view', () => {
+	cy.viewport(606,498);
```

Comment:
I promise this is the last thing - can you make this step more generic? You can just use this exact code we have in our regression repo instead: https://github.com/NCIOCPL/cgov-digital-platform-acceptance-tests/blob/master/cypress/integration/common/config.js